network-protection/network-protection-impl/src/main/java/com/duckduckgo/networkprotection/impl/notification/NetPDisabledNotificationScheduler.kt

```diff
@@ -82,7 +83,7 @@ class NetPDisabledNotificationScheduler @Inject constructor(
 
     override fun onVpnStartFailed(coroutineScope: CoroutineScope) {
         coroutineScope.launch(dispatcherProvider.io()) {
-            if (networkProtectionRepository.vpnAccessRevoked) {
+            if (!netpSubscriptionManager.getVpnStatus().isActive()) {
```

Comment:
Not 100% sure on this and I am not 100% sure how to test it either. If the user is a netp beta user and for some reason gets onVpnStartFailed, i would assume this could trigger VPN access revoked dialog to show.